docs/fixtures/readme.md

```diff
@@ -49,7 +49,7 @@ func main() {
 or for example when using the `BlobFixtureWriterFactory`:
 
 ```yaml
-blobstore:
+blob:
   blobconfig:
```

Comment:
Don't you need to add the client name here now? E.g.
```
blob:
  default:
    blobconfig:
```

Reply:
`blobconfig` is the clientname here :-D